src/mock-doc/element.ts

```diff
@@ -502,47 +502,59 @@ export class MockUListElement extends MockHTMLElement {
   }
 }
 
+type CanvasContext = '2d' | 'webgl' | 'webgl2' | 'bitmaprenderer';
+export class CanvasRenderingContext {
+  context: CanvasContext;
+  contextAttributes: any;
+  constructor(context: CanvasContext, contextAttributes?: any) {
```

Comment:
any way we an avoid the `any` here? if not, no worries

Reply:
Pushed a change to make types more strict.